content/items/rggen.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "RgGen"
+description: "Code generator for configuration and status registers"
+authors:
+  - Taichi Ishitani
+links:
+  gh: "rggen/rggen"
+  docs: "https://github.com/rggen/rggen/wiki"
+tags:
+  - Code generator
+  - CSR
+  - Configuration register
```

Comment:
Tags need to be a single word (no spaces), and are lowercased anyway. Please, replace:

- `Code generator` with `code-generator` or `code-gen` or `codegen`.
- `Configuration register` with `configuration-register`.
- `UVM reg` with `UVM-register` or `UVM-reg`.

Reply:
I fixed them.